docs/Installing.asciidoc

```diff
@@ -604,6 +604,10 @@ ongoing testing. Example:
 systemctl kill --signal SIGHUP 'openqa-worker-auto-restart@*.service'
 --------------------------------------------------------------------------------
 
+There is also the systemd unit `openqa-soft-restart-worker@.path` which invokes
+the command above whenever the worker configuration under `/etc/openqa/workers.ini`
+changes.
+
```

Comment:
We still have that list of worker service files above this section, i.e. above line 591. Should both openqa-soft-restart-worker@.{path,service} also be mentioned there? Also I think it would be good if you could make it clear to the person setting up an installation if that needs any action or is done automatically.

Reply:
I've mentioned it in the list above and that the unit is not enabled by default.